appveyor.yml

```diff
@@ -27,6 +27,7 @@ install:
   - "pip install nose-timer nibabel nitime"
   - "python setup.py develop"
   - "SET MNE_SKIP_NETWORK_TESTS=1"
+  - "SET MNE_SKIP_FLASH_CALL=1"
```

Comment:
You probably don't need this for AppVeyor actually, since it won't have FreeSurfer it won't be able to run the test right?